lib/squadster/workers/notify_attendants.ex

```diff
@@ -1,11 +1,39 @@
 defmodule Squadster.Workers.NotifyAttendants do
   use Task
 
+  import Ecto.Query
+  import HTTPoison
+
+  alias Squadster.Formations.SquadMember
+  alias Squadster.Repo
+
+  @bot_endpoint "https://mercury-dev.herokuapp.com/message"
+  @message "Завтра вы ответственный за лопату!"
```

Comment:
Probably we should store text that comes to front-end or bot in [locales](https://github.com/squadster/squadster-api/tree/master/priv/gettext), but I'm not sure how it works here, so take a look please.

Also we'll need to discuss the phrase itself